psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -222,6 +217,15 @@ def scf_initialize(self):
         core.timer_on("HF: Guess")
         self.guess()
         core.timer_off("HF: Guess")
+        # Print out initial docc/socc/etc data
```

Comment:
Interesting that this will put the pre-iterations after two-electron integrals which will change the input file substantially. Can you you post a before/after comparison?

Reply:
added a comparison.